src/components/grid-aware/ArticleSpotlightCard/ArticleSpotlightCard.module.css

```diff
@@ -0,0 +1,96 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  grid-template-rows: 600px;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedBackground {
+  grid-column: 2 / span 2;
+  grid-row: 1 / -1;
```

Comment:
Is there a reason for the -1? grid-row: 1 works perfectly fine.

Reply:
Good catch! I think this is probably a vestige from an earlier iteration of this where the grid had multiple rows, and where I wanted the background to span all rows, which is what the `-1` would have allowed it to do. I can remove the `/ -1` here because there's only a single row.

Reply:
Just fixed this, squashed it into the original commit, and did a force-push back out to this branch. LMK if this all looks good to you now.